homeassistant/components/unifiprotect/button.py

```diff
@@ -73,6 +93,19 @@ class ProtectButtonEntityDescription(
 )
 
 
+async def _remove_adopt_button(
+    hass: HomeAssistant, device: ProtectAdoptableDeviceModel
+) -> None:
+
+    entity_registry = await er.async_get_registry(hass)
```

Comment:
```suggestion
    entity_registry = er.async_get(hass)
```

`async_get_registry` is deprecated and going away in the future